tests/test_main.py

```diff
@@ -146,9 +146,9 @@ def test_mordred_sword_is_not_poisoned(base_knights_config):
 
 
 def test_arthur_armour_weak(base_knights_config):
-    base_knights_config["arthur"]["armour"][0]["protection"] -= 10
-    base_knights_config["arthur"]["armour"][1]["protection"] -= 10
-    base_knights_config["arthur"]["armour"][0]["protection"] -= 10
+    base_knights_config["arthur"]["armour"][0]["add_some_abilities"] -= 10
+    base_knights_config["arthur"]["armour"][1]["add_some_abilities"] -= 10
+    base_knights_config["arthur"]["armour"][0]["add_some_abilities"] -= 10
```

Comment:
This section of code violates the DRY principle. You are repeating the same operation on 'arthur's armour. Consider using a loop to iterate over the armour list and apply the changes.